packages/perseus/src/interactive2.ts

```diff
@@ -7,19 +7,16 @@ import KhanColors from "./util/colors";
 const Interactive2 = {
     MovablePoint: MovablePoint,
     addMovablePoint: function (graphie: any, options: any): any {
-        // @ts-expect-error [FEI-5003] - TS2350 - Only a void function can be called with the 'new' keyword.
```

Comment:
These are no longer needed now that we're explicit that the `Movable` function returns `void` and not `undefined`.